### ISSUE TITLE
[Docs]: Error on adding a theme to GitHub pages

### ISSUE THREAD

**Issue by DRMPN** [jekyll/jekyll#9437]
## Motivation

Both [Jekyll](https://jekyllrb.com/docs/themes/#installing-a-theme) and [GitHub](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll) docs have a little bit of misleading information about adding a theme to GitHub Pages.

Every aspect of this problem has already been described (with logs) in the following issue **_[Update "Add theme to Pages site" #25314](https://github.com/github/docs/issues/25314)_** in the official GitHub docs.
**TL;DR**
Each supported theme has the following syntax: `theme: jekyll-theme-<name>`, but only minima requires `theme: <name>`.

## Suggestion

I think there are two ways to fix this problem:
1) Mention in both Jekyll and GitHub docs this specific case about minima theme syntax in _config.yml;
2) Modify the code of the minima theme so that it uses the same syntax as the other themes.

**Comment by mattr-** [comment]
Sorry, but I'm not clear what this issue is asking for. Closing for now. If more clarity can be provided, we can reopen this issue
